### ISSUE TITLE
Clicking the sub-entity on mobile opens the wrong entity

### ISSUE THREAD

**Issue by davidlang42** [benct/lovelace-multiple-entity-row#253]
In a full size browser/ PC, clicking the sub-entity value opens the entity details for the sub-entity
=> This is the desired behaviour

In a mobile browser (or the mobile app), clicking the sub-entity value open the entity details for the main entity
=> This is incorrect.

This is reproducable with the following YAML configuration for lovelace card:
```
entities:
  - entity: sensor.downstairs_temperature
    type: custom:multiple-entity-row
    name: Downstairs
    secondary_info:
      entity: sensor.downstairs_battery
      name: 🔋
    entities:
      - entity: sensor.downstairs_humidity
        name: humidity
```
1. Open HA in Chrome on Android, or the Android HomeAssistant app
2. Click the humidity value
Expected: entity details for sensor.downstairs_humidity
Actual: entity details for sensor.downstairs_temperature

**Comment by mattthewclayton** [comment]
Hey, thanks for opening this issue, it was on my todo list to raise. I got a slap on the wrist for commenting on a closed ticket but it does look incredibly similar (in symptoms) to this issue: https://github.com/benct/lovelace-multiple-entity-row/issues/217 

There is a comment in that thread that they gave up looking at iOS because it seemed too buggy, someone was asked to raise a ticket for iOS in December but I could not find one.

I can confirm if I remove my "tap_action" blocks from my example, I get the same behaviour as you David the more info of the main entity is shown in iOS but it works as expected on web.

Are you also using iOS @davidlang42 ?

**Re-posting my comment from the other ticket:** 

In order for the tap_action to work on the entity (not the row) I have to do a long (3D?) press, and even then it's fairly intermittent if it works, you have to be very precise with your finger it seems.

Is this likely an issue with this integration or with the iOS companion app?

HA Version: 2022.5.0
Companion App: 2022.3

![Untitled](https://user-images.githubusercontent.com/9895197/169512112-b7ca84b2-3031-41ad-bce3-fa6e5a448c33.png)
```
entities:
  - entity: climate.living_room
    type: custom:multiple-entity-row
    icon: mdi:temperature-celsius
    secondary_info:
      attribute: current_temperature
      unit: °C
    name: Living
    tap_action: none
    show_state: false
    entities:
      - entity: light.living_room
        icon: mdi:ceiling-light
        name: Living
        tap_action:
          action: toggle
      - entity: light.dining_room
        icon: mdi:ceiling-light
        name: Dining
        tap_action:
          action: toggle
      - entity: light.kitchen
        icon: mdi:ceiling-light
        name: Kitchen
        tap_action:
          action: toggle
      - entity: switch.left_lamp
        icon: mdi:lamp
        name: Left
        tap_action:
          action: toggle
      - entity: switch.right_lamp
        icon: mdi:lamp
        name: Right
        tap_action:
          action: toggle
```

**Comment by mattthewclayton** [comment]
Just did an obvious test to try and rule out the companion app - I get the exact same behaviour using Safari and Chrome on my iOS device. It's an iPhone 11. 

I also tried on my older iPad Air 2 and its the same, although on that I can not do a long press to work around the issue. (that is not running the latest iOS, its running 14.7.1, I am just going to update it to see if it changes the behaviour).

**Comment by davidlang42** [comment]
Only Android (mobile browser and companion app), not iOS, but yes it looks similar

**Comment by mattthewclayton** [comment]
Updating my iPad Air to the latest iOS (15.5) it now behaves the same as my iPhone, long presses do trigger the correct behaviour.

**Comment by ildar170975** [comment]
> In a mobile browser (or the mobile app), clicking the sub-entity value open the entity details for the main entity

Do not confirm this.
iOS 15.4.
Tapping on a secondary entity opens more-info of this entity, tapping on a main entity opens more-info of the main entity.
[There was an issue, it was fixed.](https://github.com/benct/lovelace-multiple-entity-row/issues/217)

If another entity was specified for more-info for a secondary entity - tapping opens a more-info for this another entity.

If another entity was specified for more-info for the main entity - tapping still opens a more-info for the main entity.
[There is an issue for this](https://github.com/benct/lovelace-multiple-entity-row/issues/188).

**Comment by mattthewclayton** [comment]
I'll tuck my tail between my legs and walk away - I discovered what my issue was. 

TL;DR: I was using an old version of multiple-entity-row.

I had two installs of multiple-entity-row, one manual and one via HACS - it seems that the HACS one was being ignored, I updated my "Resources" section in my dashboards section and deleted the old version of multiple-entity-row - and after a "reset frontend cache" from the companion app - hey presto, all is working for me now.

**Comment by jpettitt** [comment]
Fixed in [4.7.0](https://github.com/benct/lovelace-multiple-entity-row/releases/tag/v4.7.0) — taps now resolve against the entity actually tapped, and no longer also fire the main row's action.
